source/includes/fundamentals/code-snippets/Monolight.java

```diff
@@ -0,0 +1,35 @@
+package fundamentals.monolightcodec;
+
+// start class
+public class Monolight {
+    private PowerStatus powerStatus = PowerStatus.OFF;
+    private Integer colorTemperature;
+
+    public Monolight() {}
+
+    // ...
```

Comment:
Nit: Are these comments needed?

Reply:
They're there for formatting the example. We use a directive called "literalinclude" which inserts all the text from a file, optionally between two comments. The "start class" and "end class" denote the start and end of the included text, and the "// ..." signifies additional but excluded code.
In the documentation example, I only wanted to show the fields and constructor, not all the getters/setters/toString methods. See the attached to see how this renders on the page:

![Screen Shot 2021-04-30 at 10 10 23 AM](https://user-images.githubusercontent.com/52428683/116707117-4efb0c80-a99c-11eb-81c0-b268b2daafe1.png)

Let me know if you have any recommended changes here.